approval_frame/backends.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.backends import ModelBackend
+from django.contrib.auth.models import User
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError, MultipleObjectsReturned
+
+class EmailOrUsernameBackend(ModelBackend):
+    """ Class to authenticate the user based on the given 
```

Comment:
I could add one after """, but aren't we following the PEP conventions (https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) ?